README.md

```diff
@@ -148,6 +156,15 @@ following attributes:
 
   * `metadata`: if `True`, [GapicMetadata](https://github.com/googleapis/googleapis/blob/master/gapic/metadata/gapic_metadata.proto) will be generated in JSON form. The default is `False`.
 
+  * `transport`: the desired transport(s) to generate, delimited by `+` e.g. `grpc+rest`.
+    * Acceptable values are `grpc` and `rest`.
+    * Defaults to `grpc`.
+
+  * `rest_numeric_enums`: if `True`, enables generation of system parameter requesting
```

Comment:
Is this implemented in the Bazel rule already?

Reply:
Yes, in this PR